aggregators/1.0/modules/ROOT/pages/aggregators-size-timeout.adoc

```diff
@@ -0,0 +1,154 @@
+= Configure Size-Based Aggregator Timeout and Max Size Fields - Mule 4
+
+To understand the Aggregators module *Size based aggregator* timeout behavior, review the following diagram that explains how the *Timeout* and *Max size* fields work. Then learn how to configure these fields in Anypoint Studio and in the XML editor in a Mule application example.
+
+Consider the following definitions:
+
+* The *Max size* field indicates the total number of elements to be aggregated before considering the aggregation complete.
+* The *Timeout* field indicates the maximum time to wait for the aggregation to complete.
+
+.Size-Based Aggregator Timeout and Max Size behavior diagram
+image::aggregator-size-timeout-diagram.png[]
+
+* The first process indicates that the aggregation is complete because the aggregated elements reached the total number of required elements before the timeout occurred.
+
+* The second process indicates that the aggregation is not complete because the aggregated elements did not reach the total number of required elements before the timeout occurred, which causes a reset of the aggregation.
```

Comment:
ditto
```suggestion
* The second process indicates that the aggregation is not complete because the total aggregated elements did not reach the number required before the timeout occurred, which causes an aggregation reset.
```